jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -1913,9 +1913,6 @@ static ClassLoader callerClassLoader() {
  *							if the library was not allowed to be loaded
  */
 static void loadLibraryWithClassLoader(String libName, ClassLoader loader) {
-	SecurityManager smngr = System.getSecurityManager();
```

Comment:
This can't be removed, there are other callers than loadLibrary(String)

Reply:
The other caller is `static void loadLibrary(Class<?> caller, String name, boolean fullPath)` which is not in `OpenJD JDK15+`, for earlier JDK level, it is invoked by `Runtime.load0() & loadLibrary0()` in which `security.checkLink(libname)` are performed first.

Reply:
See System.loadLibrary() as well.

Reply:
Right, this PR moved the security check from `loadLibraryWithClassLoader()` into `System.loadLibrary()`, note that a similar check already is within `System.load()`.